news/2025/2025-07-05-osucatch-world-cup-2025-quarterfinals-recap.md

```diff
@@ -0,0 +1,194 @@
+
+Huge Congratulations for all of them for making it this far in the tournament and we hope to you see again next year! With that being said, let's dive into some match highlights of this weekend!
+
+## Match highlights
+
+### ::{ flag=SE }:: Sweden vs. ::{ flag=AU }:: Australia
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/118526625)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2498078516?t=1h34m54s)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=1h34m54s&video=2498078516" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+To anyone who watched this past weekend's matches the Sweden (#16) vs. Australia (#22) one is for sure the highlight of the round. For the past two years, these two teams have met each other in the Round of 32, with Australia winning twice. This year however, it was in the losers' bracket of the Quarterfinals that they would meet, with the higher stakes of elimination taking their newly formed rivalry to a new level.
+
+The match started out as one would expect, with both teams trading points back and forth on their own picks. Sweden opted for [MM1](https://osu.ppy.sh/beatmapsets/2391796#fruits/5177786) and [HD3](https://osu.ppy.sh/beatmapsets/2391752#fruits/5177635), with a notable FC from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) on the latter, showing off their strong Hidden abilities. As for Australia, they knew their strength lied in Double Time and Hard Rock, and picked [DT3](https://osu.ppy.sh/beatmapsets/2391770#fruits/5177673) and [DT1](https://osu.ppy.sh/beatmapsets/2393172#fruits/5181836), winning on them comfortably. Each team knew their strengths and weaknesses, and each map was won with a clear advantage to the team picking it.
+
+What broke this dynamic was be Sweden's [HD1](https://osu.ppy.sh/beatmapsets/2391931#fruits/5178116) pick, which Australia ended up winning by only 2,286 points. Conceding a break point in a match in which both teams look really comfortable on their own picks is never good, as it can be enough to close out a match without going to the tiebreaker. But to everyone's surprise, it was now Sweden that would win on Australia's comfort Mod, on [DT2](https://osu.ppy.sh/beatmapsets/2391758#fruits/5177654). The scoreline was now even again.
```

Comment:
```suggestion
What broke this dynamic was Sweden's [HD1](https://osu.ppy.sh/beatmapsets/2391931#fruits/5178116) pick, which Australia ended up winning by only 2,286 points. Conceding a break point in a match in which both teams look really comfortable on their own picks is never good, as it can be enough to close out a match without going to the tiebreaker. But to everyone's surprise, it was now Sweden that would win on Australia's comfort Mod, on [DT2](https://osu.ppy.sh/beatmapsets/2391758#fruits/5177654). The scoreline was now even again.
```